1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -17,7 +16,14 @@ const BIRTHDAYS = [
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+  let  i = 0;
+  while(i < BIRTHDAYS[i].length){
```

Comment:
```suggestion
  while(i < BIRTHDAYS.length){
```
You want to loop through the birthday array, not just through an index in the array

Reply:
Yes. thanks